### ISSUE TITLE
I have determined install valgrind, SFFT is still configured to fail . How to solve this

### ISSUE THREAD

**Issue by lph945** [davidediger/sfft#1]
I have determined install valgrind, SFFT is still configured to fail . How to solve this

[root@node1 sfft-master]# ls
AUTHORS.txt  CHANGELOG.txt  COPYING.txt    doc       python      sfft-master  test   wscript
build        configure      COPYRIGHT.txt  Makefile  README.txt  src          tools
[root@node1 sfft-master]# ./configure
Setting top to                           : /export/home/lph/sfft-master 
Setting out to                           : /export/home/lph/sfft-master/build 
Checking for 'g++' (c++ compiler)        : /usr/bin/g++ 
Checking for program pkg-config          : /usr/bin/pkg-config 
Checking for 'valgrind'                  : not found 
The configuration failed
(complete log in /export/home/lph/sfft-master/build/config.log)

**Comment by davidediger** [comment]
Sorry but I was not able to reproduce.  You can try to post config.log.

**Comment by dnbaker** [comment]
I was able to reproduce: this report was made in slight error.

The problem is that waf assumes that one uses the PKG_CONFIG system. It doesn't check that valgrind is an available executable: it checks that there is a valid valgrind.pc. Similarly, it checks for FFTW the same way.

I have all requirements installed on my machine, but in order to get around waf failures, I would have to create *.pc files for each, which is a bizarre installation requirement.

A smarter ./configure script which actually checks for the available of headers and libraries to link against would work just fine on my machine, as it does for spiral's main release and FFTW itself.

tl;dr: Your build system only works for platforms which use the PKG_CONFIG system, which I never have across a variety of servers, workstations, and personal machines. A better system would be a classical configure script.
